admin-front/src/components/TimeSeries/TimeSeries.tsx

```diff
@@ -0,0 +1,164 @@
+  const exportData = useRef<Record<string, number>[]>([]);
+  const [resolution, setResolution] = useState<Resolution>(
+    ResolutionValues.DAY,
+  );
+  const [month, setMonth] = useState<Month>('All');
+  const [year, setYear] = useState<number>(moment().year());
+
+  const {
+    timeStamps: priceVolumeTimeStamps,
+    timeStampValues: priceVolumeTimeStampValues,
+    fetchedTimeSeries: timeSeriesPriceVolume,
+  } = useGetTimeSeriesData({
+    timeSeriesType: TimeSeriesTypeEnum.PRICE_VOLUME,
+    resolution,
+    year,
+    month,
+  });
+
+  const {
+    timeStamps: nftCountTimeStamps,
+    timeStampValues: nftCountTimeStampValues,
+    firstYearTimeSeriesRecord,
+    fetchedTimeSeries: timeSeriesNftCount,
+  } = useGetTimeSeriesData({
+    timeSeriesType: TimeSeriesTypeEnum.NFT_COUNT,
+    resolution,
+    year,
+    month,
+  });
+
+  useEffect(() => {
+    if (timeSeriesPriceVolume?.length && timeSeriesNftCount?.length) {
+      const dataForExport = getDataForExport([
+        {
+          timeseries: destroyReferenceAndClone(timeSeriesPriceVolume),
+          context: TimeSeriesTypeEnum.PRICE_VOLUME,
+        },
+        {
+          timeseries: destroyReferenceAndClone(timeSeriesNftCount),
+          context: TimeSeriesTypeEnum.NFT_COUNT,
+        },
+      ]);
+      exportData.current = dataForExport;
+    }
+  }, [timeSeriesNftCount, timeSeriesPriceVolume]);
+
+  const handleResolutionChange = (event: MouseEvent<HTMLElement>) => {
+    const newResolution = (event.target as HTMLInputElement)
+      .value as Resolution;
+    setResolution(newResolution);
+  };
+
+  const handleMonthChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const newMonth = event.target.value as Month;
+    setMonth(newMonth);
+  };
+
+  const handleYearChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const newYear = event.target.value as unknown as number;
+    setYear(newYear);
+  };
+
+  const exportCsvOptions = {
+    fieldSeparator: ',',
+    filename: 'Analytics',
+    showLabels: true,
+    useTextFile: false,
+    useBom: true,
+    useKeysAsHeaders: true,
+  };
+
+  const csvExporter = new ExportToCsv(exportCsvOptions);
+
+  return (
+    <Grid container direction="column" spacing={2} wrap={'nowrap'}>
+      <Grid container justifyContent={'space-between'} gap={1} item>
+        <Grid item>
+          <Paper sx={{ padding: '20px' }}>
+            <TimeSeriesFilter
+              resolution={resolution}
+              year={year}
+              month={month}
+              firstYearTimeSeriesRecord={firstYearTimeSeriesRecord}
+              handleResolutionChange={handleResolutionChange}
+              handleMonthChange={handleMonthChange}
+              handleYearChange={handleYearChange}
+            />
+          </Paper>
+        </Grid>
+        <Grid item>
+          <Button
+            variant="outlined"
+            sx={buttonStyles}
+            onClick={() => csvExporter.generateCsv(exportData.current)}
```

Comment:
The export functionality seems to have some issues with the timestamps. No matter what resolution, there's always an HH:MM component that counts upwards with each datapoint.